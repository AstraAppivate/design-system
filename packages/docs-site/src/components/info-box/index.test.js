import React from 'react'
import renderer from 'react-test-renderer'

import InfoBox from './index'

/* eslint-disable react/no-children-prop */

describe('Info Box', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<InfoBox children={['Content goes here']} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
