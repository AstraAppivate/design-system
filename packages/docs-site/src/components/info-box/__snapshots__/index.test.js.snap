// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Box renders correctly 1`] = `
<a
  className="info-box  link"
  href={null}
>
  Content goes here
</a>
`;
